app/code/Ometria/Api/Controller/V1/Products.php

```diff
@@ -262,16 +262,14 @@ protected function serializeItem($item)
                 ->addFieldToFilter('attribute_code', $attribute['attribute_code'])
                 ->getFirstItem();
 
-            $key = 'value';
-
-            if (in_array($full_attribute->getFrontendInput(), ['select', 'multiselect'])) {
-                $key = 'id';
-            }
+            $inputType = $fullAttribute->getFrontendInput();
+            $type = $inputType == 'multiselect' ? '&' . $attribute['attribute_code'] : $attribute['attribute_code'];
+            $valueIdx = in_array($inputType, ['select', 'multiselect']) ? 'id' : 'value';
 
             $tmp['attributes'][] = [
-                'type'  => $attribute['attribute_code'],
-                $key    => $attribute['value'],
-                'label' => $full_attribute->getFrontendLabel()
+                'type'    => $type,
+                $valueIdx => $attribute['value'],
+                'label'   => $fullAttribute->getFrontendLabel()
```

Comment:
Or just `$inputType`

Reply:
$inputType here will be the attribute's type (select, multiselect etc), whereas the 'label' field needs to show the attribute's label, such as "Colour". The difference being the call to getFrontendInput() versus getFrontendLabel() on the attribute. They do look very similar!